components/LogoNav/LogoNav.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import LogoNav from './LogoNav';
+
+describe('LogoNav Component', () => {
+  it('renders the logo and links to the /league/all page', () => {
+    render(<LogoNav />);
+
+    const logoImage = screen.getByAltText('Gridiron Survivor logo');
```

Comment:
Never grab a component by the alt text. This could change any time, thus failing your test very easily. Add a `data-testid`, then find the component by this attribute.

Reply:
Updated to grab by `getByTestId` instead.